lib/std/rand/Romu192.zig

```diff
@@ -0,0 +1,141 @@
+// Website: romu-random.org
+// Reference paper:   http://arxiv.org/abs/2002.11331
+
+// important note: this RNG exposes all of it's state in the random sequences generated, so it is trivial to write a state prediction function.
+// it's use is recommended only in context where there is no risk of prediction, e.g. it is perfectly safe for a monte carlo simulation.
+
+const std = @import("std");
+const Random = std.rand.Random;
+const math = std.math;
+const Romu192 = @This();
+
+xState: u64,
+yState: u64,
+zState: u64, // set to nonzero seed
```

Comment:
Some simple style guide changes should probably be made. See [the documentation about this](https://ziglang.org/documentation/master/#Names).
```suggestion
x_state: u64,
y_state: u64,
z_state: u64, // set to nonzero seed
```